[2024-08-25T03:00:44.760+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: json_FT_processing_pipeline_03.bulk_load_task scheduled__2024-08-23T08:00:00+00:00 [queued]>
[2024-08-25T03:00:44.770+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: json_FT_processing_pipeline_03.bulk_load_task scheduled__2024-08-23T08:00:00+00:00 [queued]>
[2024-08-25T03:00:44.770+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 61
[2024-08-25T03:00:44.783+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): bulk_load_task> on 2024-08-23 08:00:00+00:00
[2024-08-25T03:00:44.789+0000] {standard_task_runner.py:60} INFO - Started process 48858 to run task
[2024-08-25T03:00:44.792+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'json_FT_processing_pipeline_03', 'bulk_load_task', 'scheduled__2024-08-23T08:00:00+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/FranceTravail_pipeline_dag03.py', '--cfg-path', '/tmp/tmplhyopz0k']
[2024-08-25T03:00:44.793+0000] {standard_task_runner.py:88} INFO - Job 54: Subtask bulk_load_task
[2024-08-25T03:00:44.806+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-08-25T03:00:44.841+0000] {task_command.py:423} INFO - Running <TaskInstance: json_FT_processing_pipeline_03.bulk_load_task scheduled__2024-08-23T08:00:00+00:00 [running]> on host 9e648299db30
[2024-08-25T03:00:44.920+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='json_FT_processing_pipeline_03' AIRFLOW_CTX_TASK_ID='bulk_load_task' AIRFLOW_CTX_EXECUTION_DATE='2024-08-23T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-23T08:00:00+00:00'
[2024-08-25T03:00:45.498+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-08-25T03:00:45.509+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=json_FT_processing_pipeline_03, task_id=bulk_load_task, execution_date=20240823T080000, start_date=20240825T030044, end_date=20240825T030045
[2024-08-25T03:00:45.569+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-08-25T03:00:45.585+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
